src/game/client/gameclient.cpp

```diff
@@ -602,13 +602,11 @@ void CGameClient::OnMessage(int MsgId, CUnpacker *pUnpacker)
 				break;
 			case GAMEMSG_TEAM_BALANCE_VICTIM:
 				{
-					const char *pMsg;
+					const char *pMsg = "";
```

Comment:
That looks like inappropriate error handling. Better ignore the message if it contains invalid data instead of trying to continue.